.gitignore

```diff
@@ -14,4 +14,5 @@ node-*
 /plugins
 /lib/coder-cloud-agent
 .home
-coverage
\ No newline at end of file
+coverage
+**/.DS_Store
```

Comment:
Sneaky `.gitignore` change!
Not sure if you meant to include this in this PR.

Reply:
Well, I keep running into that error (thanks macOS lol). So if it's okay, I would like to include it!

Reply:
Sounds good then!